src/api/image/src/routes/image.js

```diff
@@ -78,6 +80,13 @@ router.use(
       return;
     }
 
+    // Return a specific photo at an index
+    if (!isNaN(image)) {
+      req.imageFilename = getPhotoAt(image);
```

Comment:
Let's throw away fractional components so we have a pure integer: `getPhotoAt(image|0);`

Reply:
Thanks, I forgot about this.